storageframework/src/tests/thread/taskthreadtest.cpp

```diff
@@ -50,19 +38,17 @@ TaskThreadTest::testNormalUsage()
     t.addTask(Task("b", 3));
     t.addTask(Task("c", 8));
     t.addTask(Task("d", 4));
-    CPPUNIT_ASSERT(t.empty()); // Still empty before critical tick has run
+    EXPECT_TRUE(t.empty()); // Still empty before critical tick has run
     dynamic_cast<TickingThread&>(t).doCriticalTick(0);
-    CPPUNIT_ASSERT(!t.empty());
-    CPPUNIT_ASSERT_EQUAL(3, (int) t.peek().getPriority());
+    EXPECT_TRUE(!t.empty());
```

Comment:
This should probably be `ASSERT_TRUE` since the line below expects the task set to be non-empty.

Reply:
Fixed